.github/workflows/mac-artifact-jfrog.yml

```diff
@@ -0,0 +1,123 @@
+name: Mac Artifact
```

Comment:
why doesn't asconfig have a mac-artifact-jfrog workflow? Is it's mac build done differently?